### PR TITLE
test: Add test for WsConnection for ping response

### PR DESCRIPTION
**Pull request by kuznetsss**

### CODE REVIEW

**Comment by kuznetsss** [comment]
Adding a test decreases code coverage 😄

File: tests/unit/util/requests/WsConnectionTests.cpp [hunk 1]
```diff
@@ -318,6 +321,40 @@ TEST_F(WsConnectionTests, MultipleConnections)
...
+    asio::spawn(ctx, [&](asio::yield_context yield) {
+        auto serverConnection = unwrap(server.acceptConnection(yield));
+
+        testing::StrictMock<testing::MockFunction<void(boost::beast::websocket::frame_type, std::string_view)>>
+            controlFrameCallback;
+        serverConnection.setControlFrameCallback(controlFrameCallback.AsStdFunction());
+        EXPECT_CALL(controlFrameCallback, Call(boost::beast::websocket::frame_type::pong, testing::_)).WillOnce([&]() {
+            serverConnection.resetControlFrameCallback();
+            asio::spawn(ctx, [&](asio::yield_context yield) {
+                auto maybeError = serverConnection.send("got ping", yield);
```
Comment (cindyyan317) [inline tests/unit/util/requests/WsConnectionTests.cpp]: Should it be "got a pong?"

**Review by cindyyan317 — DISMISSED** [review]
LGTM
I think this test is to ensure our ws client is able to see pong, right?

**Review by cindyyan317 — APPROVED** [review]
LGTM

**Pull request merged**